code/modules/vehicles/secway.dm

```diff
@@ -7,6 +7,9 @@
 	armor = list("melee" = 10, "bullet" = 0, "laser" = 10, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 60, "acid" = 60)
 	key_type = /obj/item/key/security
 	integrity_failure = 0.5
+	var/last_rev
+
+	var/obj/item/reagent_containers/food/snacks/grown/banana/eddie_murphy
```

Comment:
please make this var less vague, and autodoc what it does with `/// if it has a banana in the pipe, it does x`

Reply:
it's fine, just needs a comment